snuba/settings/settings_distributed.py

```diff
@@ -9,6 +9,7 @@
         "database": os.environ.get("CLICKHOUSE_DATABASE", "default"),
         "http_port": int(os.environ.get("CLICKHOUSE_HTTP_PORT", 8123)),
         "storage_sets": {
+            "cdc",
```

Comment:
I think we may have a push safety problem with this validation in prod
https://github.com/getsentry/snuba/blob/master/snuba/clusters/cluster.py#L315-L323
The code that defines the new storage set will hit production before the config change which does not know the storage set.

Reply:
Not sure what you mean exactly. I believe the new storage set needs to be added to prod before the code can be deployed. Once it is deployed the storage set must be in the settings otherwise Snuba will raise an error.

Reply:
right, that is the point I was talking about. If you already have a PR for the prod config that is ok.

Reply:
Should be linked to this PR